### PR TITLE
parameter container refactor part1

### PR DESCRIPTION
**Pull request by jd-lara**
Current tests disabled 

const DISABLED_TEST_FILES = [
    "test_device_hydro_generation_constructors", @sourabhdalvi will fix energy formulations 
    "test_operations_problem", Dual retrieve fails 
    "test_simulation_build.jl",
    "test_simulation_execute.jl",
    "test_simulation_store.jl",
    "test_simulation_results.jl",
    "test_simulation_results_export.jl",
]

### CODE REVIEW

File: src/devices_models/devices/electric_loads.jl [hunk 1]
```diff
@@ -153,32 +151,31 @@ function DeviceRangeConstraintSpec(
...
 ) where {T <: PSY.ElectricLoad}
     return NodalExpressionSpec(
-        "max_active_power",
-        "Q",
+        parameter,
+        T,
         use_forecasts ? x -> PSY.get_max_reactive_power(x) : x -> PSY.get_reactive_power(x),
         -1.0,
-        T,
+        :nodal_balance_reactive,
```
Comment (daniel-thom) [inline src/devices_models/devices/electric_loads.jl:164]: Do you plan to make changes to this free-form symbol?

Comment (jd-lara) [reply src/devices_models/devices/electric_loads.jl:164]: This will change in an upcoming PR to refactor expression containers.

**Review by daniel-thom — APPROVED** [review]
LGTM

File: src/core/optimization_container.jl [hunk 2]
```diff
@@ -47,6 +47,45 @@ mutable struct OptimizationContainer <: AbstractModelContainer
     end
 end
 
+model_time_steps(optimization_container::OptimizationContainer) =
+    optimization_container.time_steps
+model_resolution(optimization_container::OptimizationContainer) =
```
Comment (claytonpbarrows) [inline src/core/optimization_container.jl:52]: why not `get_model_resolution`?

File: src/core/optimization_container.jl [hunk 3]
```diff
@@ -47,6 +47,45 @@ mutable struct OptimizationContainer <: AbstractModelContainer
     end
 end
 
+model_time_steps(optimization_container::OptimizationContainer) =
```
Comment (claytonpbarrows) [inline src/core/optimization_container.jl:50]: why not `get_model_time_steps`?

**Pull request merged**